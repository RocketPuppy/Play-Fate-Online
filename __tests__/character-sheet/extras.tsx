import * as React from "react";
import * as renderer from "react-test-renderer";
import Extras from "../../src/character-sheet/extras";

test("renders all the pieces", () => {
  const render = renderer.create(<Extras />);

  expect(render.toJSON()).toMatchSnapshot();
});
