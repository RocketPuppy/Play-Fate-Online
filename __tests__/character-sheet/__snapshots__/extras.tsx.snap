// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Extras
  </h2>
  <form>
    <div>
      <div>
        <label>
          <span>
            Name
          </span>
          <input
            defaultValue=""
            onChange={[Function]}
            placeholder="Izanagi"
            type="text"
          />
        </label>
        <label>
          <span>
            Description
          </span>
          <textarea
            onChange={[Function]}
            placeholder="Izanagi is available to help out during conflicts"
            value=""
          />
        </label>
        <button
          disabled={true}
          onClick={[Function]}
        >
          Add new extra
        </button>
      </div>
    </div>
  </form>
</div>
`;
